File: MODUL.6/PRAK601_2310817120010_NurHikmah.c
#include <stdio.h>

int main()
{
    int baris, kolom;

    printf("Input\n");
    scanf("%d %d", &baris, &kolom);

    int matriks[baris][kolom];
    for (int i = 0; i < baris; i++)
    {
        for (int j = 0; j < kolom; j++)
        {
            scanf("%d", &matriks[i][j]);
        }
    }

    // Menampilkan matriks
    printf("\nOutput\n");
    for (int i = 0; i < baris; i++)
    {
        for (int j = 0; j < kolom; j++)
        {
            printf("%d ", matriks[i][j]);
        }
        printf("\n");
    }
    return 0;
}

File: MODUL.6/PRAK601_2310817120010_NurHikmah.py
print("Input")
baris, kolom = map(int, input().split())

matriks = []
input_nilai = list(map(int, input().split()))

index = 0
for i in range(baris):
    row = []
    for j in range(kolom):
        row.append(input_nilai[index])
        index += 1
    matriks.append(row)

print("\nOutput")
for i in range(baris):
    for j in range(kolom):
        print(matriks[i][j], end=" ")
    print()
